proto/data.go

```diff
@@ -321,13 +321,24 @@ func (v *Value) Verify(key []byte) error {
 				cksum, Key(key), v)
 		}
 	}
-	if v.Bytes != nil && v.Integer != nil {
-		return util.Errorf("both the value byte slice and integer fields are set for key %s: [% x]",
-			Key(key), v)
-	}
 	return nil
 }
 
+// SetInteger encodes the specified int64 value into the bytes field of the
+// receiver.
+func (v *Value) SetInteger(i int64) {
```

Comment:
Eh, it depends. If the receiver type is usually a pointer in practice, I'd agree. Otherwise, you're just losing out on the compiler-provided nil check by going this route.